atorch/README.md

```diff
@@ -1,25 +1,114 @@
-# ATorch Introduction
+# ATorch
+<div id="top" align="center">
 
-ATorch is an extension library of PyTorch developed by Ant Financial. It provides efficient and easy-to-use model training and optimization capabilities. The design principle is to minimally invade the native PyTorch programming style, and through API interfaces, provide performance optimizations in aspects such as IO/preprocessing, computation, and communication (including automatic optimization), as well as support for large model training.
+   <img src="docs/img/atorch.png" alt="Editor" width="500">
 
+   ATorch: Making LLMs more efficient and reproducible for everyone.
```

Comment:
LLM efficient and reproducible means make language model (model structure) efficient.

efficient and reproducible should refer to training.